v1/index.js

```diff
@@ -22,5 +22,7 @@ module.exports = {
   BinaryHTTPEmitter,
   StructuredHTTPReceiver,
   BinaryHTTPReceiver,
+  Cloudevent: event,
```

Comment:
Can we not add `Cloudevent` and just use `CloudEvent`? There is no reason for an alias.